indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -150,64 +165,66 @@ export default function FormItem({
     ) : null;
 
   return (
-    <div
-      styleName={`form-item ${toClasses({
-        'disabled': !isEnabled || paidItemLocked,
-        'purged-disabled': showPurged,
-        'paid-disabled': !showPurged && paidItemLocked,
-        'editable': setupMode,
-      })}`}
-    >
-      {sortHandle}
-      <div styleName="content">
-        {InputComponent ? (
-          meta.customFormItem ? (
-            <InputComponent
-              showAsRequired={showAsRequired}
-              isRequired={inputRequired}
-              disabled={disabled}
-              isPurged={showPurged}
-              retentionPeriodIcon={retentionPeriodIcon}
-              htmlId={htmlId}
-              {...inputProps}
-            />
+    show && (
```

Comment:
`if (!show) return null;`

That way you also avoid the huge change in the diff here which make this change here hard to review (w/o hiding whitespace changes) :)